test/int/dbuser_test.go

```diff
@@ -148,27 +148,31 @@ var _ = Describe("AtlasDatabaseUser", Label("int", "AtlasDatabaseUser"), func()
 		return kube.NormalizeIdentifier(createdProject.Spec.Name) + suffix
 	}
 
+	byCreatingDefaultAWSandAzureClusters := func() {
+		By("Creating clusters", func() {
+			createdClusterAWS = mdbv1.DefaultAWSCluster(namespace.Name, createdProject.Name).Lightweight()
+			Expect(k8sClient.Create(context.Background(), createdClusterAWS)).ToNot(HaveOccurred())
+
+			createdClusterAzure = mdbv1.DefaultAzureCluster(namespace.Name, createdProject.Name).Lightweight()
+			Expect(k8sClient.Create(context.Background(), createdClusterAzure)).ToNot(HaveOccurred())
+
+			Eventually(
+				func(g Gomega) {
+					success := testutil.WaitFor(k8sClient, createdClusterAWS, status.TrueCondition(status.ReadyType), validateClusterCreatingFuncGContext(g))()
+					g.Expect(success).To(BeTrue())
+				}).WithTimeout(ClusterUpdateTimeout).WithPolling(interval).Should(Succeed())
+
+			Eventually(
+				func(g Gomega) {
+					success := testutil.WaitFor(k8sClient, createdClusterAzure, status.TrueCondition(status.ReadyType), validateClusterCreatingFuncGContext(g))()
+					g.Expect(success).To(BeTrue())
+				}).WithTimeout(ClusterUpdateTimeout).WithPolling(interval).Should(Succeed())
+		})
+	}
+
 	Describe("Create/Update two users, two clusters", func() {
 		It("They should be created successfully", func() {
-			By("Creating clusters", func() {
```

Comment:
The linter made me do this despite me not planning to change this file in any way.